editor/export/project_export.cpp

```diff
@@ -1426,7 +1426,15 @@ void ProjectExportDialog::_export_project() {
 	}
 
 	if (!current->get_export_path().is_empty()) {
-		export_project->set_current_path(current->get_export_path());
+		String path = current->get_export_path();
+		if (platform->get_name() == "Android") {
```

Comment:
Let's check the class type of `platform` and ensure it's `EditorExportPlatformAndroid`. This will ensure any platform type derived from `EditorExportPlatformAndroid` gets the same treatment.